noteable_magics/sql/meta_commands.py

```diff
@@ -399,21 +400,22 @@ def run(self, invoked_as: str, args: List[str]) -> Tuple[DataFrame, bool]:
 
         inspector = self.get_inspector()
 
-        is_view = relation_name in inspector.get_view_names(schema)
-
-        if not is_view:
```

Comment:
This old way of determining if relation existed didn't work with BigQuery. Had to rephrase to work with both the existing dialects plus BigQuery.